src/ui-test/tests/camel.settings.test.ts

```diff
@@ -0,0 +1,261 @@
+
+            contentAssist = await editor.toggleContentAssist(true) as ContentAssist;
+            const items = await contentAssist.getItems();
+
+            if (proposalAvailable) {
+                assert.equal(items.length, proposalsCount);
+                const proposalItem = await contentAssist.getItem(proposal);
+                assert.equal(await getTextExt(proposalItem), proposal);
+            } else {
+                assert.equal(items.length, 0);
+            }
+        }
+    });
+
+    describe('Camel catalog user preference version set', function () {
+        this.timeout(90000);
+
+        before(_setup());
+
+        // reset version to default
+        async function _setDefaultCamelCatalogVersion() {
+            resetUserSettings(CATALOG_VERSION_ID);
+        }
+
+        describe('Different catalog versions', function () {
+
+            after(_setDefaultCamelCatalogVersion);
+
+            it('Default version', async function () {
+                // open file
+                await VSBrowser.instance.openResources(path.join(RESOURCES, CAMEL_CONTEXT_XML));
+
+                // add component
+                const editor = new TextEditor();
+                await editor.isDisplayed();
+                await editor.typeTextAt(3, XML_URI_POSITION, 'file-watch');
+
+                // open ca
+                contentAssist = await editor.toggleContentAssist(true) as ContentAssist;
+                const items = await contentAssist.getItems();
+                // should be 1
+                assert.equal(items.length, 1);
+
+                // check if content is expected
+                const expectedContentAssist = 'file-watch:path';
+                const timer = await contentAssist.getItem(expectedContentAssist);
+                assert.equal(await getTextExt(timer), expectedContentAssist);
+
+                // close file w\o saving
+                await closeEditor(CAMEL_CONTEXT_XML, false);
+            });
+
+            it('2.15.1 version', async function () {
+                // set version
+                await setCamelCatalogVersion('2.15.1');
+
+                // open file
+                await VSBrowser.instance.openResources(path.join(RESOURCES, CAMEL_CONTEXT_XML));
+
+                // add component
+                const editor = new TextEditor();
+                await editor.isDisplayed();
+                await editor.typeTextAt(3, XML_URI_POSITION, 'file-watch');
+
+                // open ca
+                contentAssist = await editor.toggleContentAssist(true) as ContentAssist;
+                const items = await contentAssist.getItems();
+                // should be empty
+                assert.equal(items.length, 0);
+
+                // close file w\o saving
+                await closeEditor(CAMEL_CONTEXT_XML, false);
+            });
+
+        });
+    });
+
+    describe('JBang user preference version set', function () {
```

Comment:
because we are under `User preferences` header (describe) it would be sufficient to have label just simply `Camel JBang version`